commands/config.go

```diff
@@ -123,7 +124,12 @@ func (cmd *configCommand) args() (res []string, switchedToTLS bool) {
 		res = append(res, fmt.Sprintf("--paths-base=%s", cmd.BasePath))
 	}
 
+	if cmd.AgentPassword != "" {
+		res = append(res, fmt.Sprintf("--agent-password=%q", cmd.AgentPassword))
```

Comment:
how about a password that includes`"`?

Reply:
another question logs, can it be logged somewhere?

Reply:
it get's logged 
![image](https://user-images.githubusercontent.com/91831719/165399874-f8ff91a3-6708-45f5-960f-130d65dc03ae.png)

was running with `./bin/pmm-admin --debug config --agent-password="my password123" --force`

Reply:
will be resolved in scope of another PR